src/app/modules/admin/components/profile-content/profile-content.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, NgForm, Validators } from '@angular/forms';
+import { DomSanitizer } from '@angular/platform-browser';
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { AppState } from 'src/app/store/app.state';
+import {
+    loadAdminInfo,
+    loadProfileAvatar,
+    removeProfileAvatar,
+} from '../../state/admin.actions';
+import { selectAvatar } from '../../state/admin.selectors';
+
+@Component({
+    selector: 'app-profile-content',
+    templateUrl: './profile-content.component.html',
+    styleUrls: ['./profile-content.component.scss'],
+})
+export class ProfileContentComponent implements OnInit {
+    panelOpenState = false;
+    fileUrl: string = '';
+    dataObj: any;
+    avatar$: Observable<any>;
+    trustUrl: any;
+    profileAvatar: string;
+    form: FormGroup;
+
+    constructor(
+        private store: Store<AppState>,
+        private sanitazer: DomSanitizer,
+    ) {}
+
+    ngOnInit(): void {
+        this.avatar$ = this.store.pipe(select(selectAvatar));
+        this.avatar$.subscribe(res => {
```

Comment:
memory leak, so u need to do:
1) create - readonly unsubscribe$ = new Subject<void>();
2) implement OnDestroy and do 
 ngOnDestroy(): void {
        this.unsubscribe$.next();
        this.unsubscribe$.complete();
        }